release notes/v0.3.0.md

```diff
@@ -0,0 +1,64 @@
+xk6-browser v0.3.0 is here! :tada:
+
+This is a minor release with some important bug fixes, a couple of new features, improvements and code refactors!
+
+
+## New features
+
+- Implemented `Browser.on(event)`. ([#268](https://github.com/grafana/xk6-browser/pull/268), [#287](https://github.com/grafana/xk6-browser/pull/287))
+
+  `on('disconnected')` returns a `Promise` you can wait on to get notified when the WebSocket connection to the browser is closed. See [this example](https://github.com/grafana/xk6-browser/blob/v0.3.0/examples/browser_on.js) for details.
+
+  This is the first async method in xk6-browser, using the newly released [event loop support in k6](https://github.com/grafana/k6/pull/2228), and marks the start of our transition to async APIs. :tada:
+
+- Exposed preconfigured mobile device emulation settings. ([#289](https://github.com/grafana/xk6-browser/pull/289))
+
+  We maintain an internal list of popular devices with their viewport size, user agent and scaling factor, which is useful to test how a web site looks and behaves on different devices. See [this example](https://github.com/grafana/xk6-browser/blob/v0.3.0/examples/device_emulation.js) for details.
+
+
```

Comment:
```suggestion
- Added the `executablePath` option. ([#246](https://github.com/grafana/xk6-browser/pull/246))

  You can now run a Chrome browser of your choice by providing the file path.
```

Reply:
This is more of a fix, no? The option was already there, it just didn't work before.

Reply:
Many options and features are there, but they don't work :( Both are fine with me (feature or bug).

Reply:
It's a feature that didn't work before, but now does, so a fix is more appropriate. Please mention it there.

Reply:
@imiric, 👍, does it make more sense now?

```suggestion
- Added the `executablePath` option. ([#246](https://github.com/grafana/xk6-browser/pull/246))

  Although you could fill this option out, it wasn't working before, and we were using a predefined path by default. We implemented the option code, and you can now run a Chrome browser of your choice by providing the file path.
```

Reply:
Yes, as a fix.

Reply:
See: bbfe45

Reply:
See: https://github.com/grafana/xk6-browser/pull/265/commits/bbfe4532fc39466dc7103006a93a07b43fc49ebd